Resources/config/core.xml

```diff
@@ -34,11 +34,21 @@
         </service>
 
         <!-- Services used to format the label, default is sonata.admin.label.strategy.noop -->
-        <service id="sonata.admin.label.strategy.bc" class="Sonata\AdminBundle\Translator\BCLabelTranslatorStrategy" />
-        <service id="sonata.admin.label.strategy.native" class="Sonata\AdminBundle\Translator\NativeLabelTranslatorStrategy" />
-        <service id="sonata.admin.label.strategy.noop" class="Sonata\AdminBundle\Translator\NoopLabelTranslatorStrategy" />
-        <service id="sonata.admin.label.strategy.underscore" class="Sonata\AdminBundle\Translator\UnderscoreLabelTranslatorStrategy" />
-        <service id="sonata.admin.label.strategy.form_component" class="Sonata\AdminBundle\Translator\FormLabelTranslatorStrategy" />
+        <service id="sonata.admin.label.strategy.bc"
+            class="Sonata\AdminBundle\Translator\BCLabelTranslatorStrategy"
+            public="false" />
```

Comment:
I found service redefinition on Media and Classification:

```bash
sullivan@sweetnexy:~/projects/fork/sonata-project$ grep -R sonata.admin.label.strategy.native .
./SonataClassificationBundle/vendor/sonata-project/admin-bundle/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php:            'label_translator_strategy' => 'sonata.admin.label.strategy.native',
./SonataClassificationBundle/vendor/sonata-project/admin-bundle/Resources/config/core.xml:        <service id="sonata.admin.label.strategy.native" class="Sonata\AdminBundle\Translator\NativeLabelTranslatorStrategy" />
# ...
./SonataMediaBundle/vendor/sonata-project/admin-bundle/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php:            'label_translator_strategy' => 'sonata.admin.label.strategy.native',
./SonataMediaBundle/vendor/sonata-project/admin-bundle/Resources/config/core.xml:        <service id="sonata.admin.label.strategy.native" class="Sonata\AdminBundle\Translator\NativeLabelTranslatorStrategy" />
```

Do you know why? Would not be better to keep it public and remove definition from these bundles?